### DISCUSSION TITLE
Hide (or shrink) little drop down arrow in md-select

### DISCUSSION THREAD

**Discussion by benfredwells** [material-components/material-web#5337] [Q&A]
Is there a way to hide the trailing icon / drop down arrow in md-outlined-select? Or at least minimize its width?

I'm trying to make selects take up less horizontal space. I've found a way to minimize most of the padding (see below and also overriding min-width on md-outlined-slider) but the arrow is still pretty wide.

  --text-field-padding: 8px;
  --md-outlined-field-bottom-space: var(--text-field-padding);
  --md-outlined-field-leading-space: var(--text-field-padding);
  --md-outlined-field-top-space: var(--text-field-padding);
  --md-outlined-field-trailing-space: var(--text-field-padding);
  --md-outlined-field-supporting-text-leading-space: var(--text-field-padding);
  --md-outlined-field-supporting-text-trailing-space: var(--text-field-padding);
  --md-outlined-select-text-field-container-shape: var(--text-field-padding);
  --md-menu-container-shape: var(--text-field-padding);

**Comment by asyncliz** [comment]
Does `--md-outlined-select-text-field-trailing-icon-size` work?

There should be a token for it. I think you should also be able to add an empty `<div slot="trailing-icon">` to remove it.

There is a link to all select tokens available in the select doc.

**Reply by MrDrone-ru** [reply]
I think the `.end` section width of `md-select`  issue is still relevant. I also failed to find the token that is responsible for the width of this section. `-trailing-icon-size` changes size of arrow (only to the smaller side), but it doesn't affect the section width unfortunately.
